packages/zowe-explorer/src/utils/LoggerUtils.ts

```diff
@@ -29,12 +29,22 @@ export class ZoweLogger {
     public static zeOutputChannel: vscode.OutputChannel;
     private static defaultLogLevel: "INFO";
     private static zeLogLevel: string;
+    private static displayName: string;
+    private static zeVersion: string;
 
     public static async initializeZoweLogger(context: vscode.ExtensionContext): Promise<void> {
         try {
             const logsPath: string = ZoweVsCodeExtension.customLoggingPath ?? context.extensionPath;
             globals.initLogger(logsPath);
-            await this.initVscLogger(context, logsPath);
+            await this.initVscLogger(logsPath);
+            if (context.extension?.packageJSON) {
+                this.displayName = context.extension.packageJSON.displayName;
+                this.zeVersion = context.extension.packageJSON.version;
+            } else {
+                const extInfo = vscode.extensions.getExtension("zowe.vscode-extension-for-zowe");
+                this.displayName = extInfo.packageJSON.displayName;
+                this.zeVersion = extInfo.packageJSON.version;
+            }
```

Comment:
Might be able to save a few lines with
`const extInfo = context.extension ?? vscode.extensions.getExtension(...);`
No need to change this though. 😋

Reply:
I like this better - no need for the `if` statement 😋 
Addressed in f40a368